apps/src/levelbuilder/ai-iteration-tools/ai-tutor/AITutorTester.tsx

```diff
@@ -18,7 +18,6 @@ interface AIInteraction {
   studentInput: string;
   systemPrompt?: string | undefined;
   levelId?: number | undefined;
-  temperature?: number | undefined;
```

Comment:
Not really related to the main changes in this PR I just noticed this un-used field, 🗑️